### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by jpnurmi** [canonical/ubuntu-desktop-installer#772]
- Semantic labels for screen reading
- Some accessibility-related system settings are not respected:
  - ~~high contrast~~
  - ~~enable animations~~
  - cursor blinking ([hard-coded to 500ms](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/editable_text.dart#L52) in Flutter)
  - double-click delay ([hard-coded to 300ms](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/gestures/constants.dart#L31) in Flutter)

~~**NOTE**: Flutter is also missing implementations for some important accessibility interfaces such as [AtkText](https://gnome.pages.gitlab.gnome.org/atk/AtkText.html), [AtkEditableText](https://gnome.pages.gitlab.gnome.org/atk/AtkEditableText.html), [AtkValue](https://gnome.pages.gitlab.gnome.org/atk/AtkValue.html) that make it possible for the screen reader to interpret text input and sliders.~~

**Comment by Feichtmeier** [comment]
0.2.5 released with the hc themes

**Comment by jpnurmi** [comment]
- https://github.com/ubuntu/yaru.dart/pull/144
- https://github.com/flutter/engine/pull/32454
- https://github.com/flutter/engine/pull/32487
- https://github.com/flutter/engine/pull/32499
- https://github.com/flutter/engine/pull/33179
- https://github.com/flutter/engine/pull/33313

**Comment by jpnurmi** [comment]
It would be nice to make accessibility settings more discoverable in the live session:
```
gsettings set org.gnome.desktop.a11y always-show-universal-access-status true
```

**Comment by jpnurmi** [comment]
Superseded by the tickets from the accessibility audit:
- https://github.com/canonical/ubuntu-desktop-installer/issues/1570
- https://github.com/canonical/ubuntu-desktop-installer/issues/1571
- https://github.com/canonical/ubuntu-desktop-installer/issues/1572
